scripts/remove-site/src/removeSite.ts

```diff
@@ -0,0 +1,45 @@
+import { exec } from "child_process";
+import * as fs from "fs";
+
+function removeFilesAndFolders() {
+    const cwd = process.cwd();
+
+    exec(`rm -rf ${cwd}/site`, (error, stdout, stderr) => {
+        if (error) {
+            console.error("Error:", error.message);
+            return;
+        }
+        if (stderr) {
+            console.error(stderr);
+            return;
+        }
+        console.log("Successfully deleted site!");
+    });
+}
+
+function removeSiteReferences() {
+    removeReference(".vscode/settings.json", /, "site"/g);
+    removeReference("admin/.env", /.*[Ss][Ii][Tt][Ee].*\n/gm);
+    removeReference("api/.env", /.*[Ss][Ii][Tt][Ee].*\n/gm);
+    removeReference(".env", /.*[Ss][Ii][Tt][Ee].*\n/gm);
+    removeReference("install.sh", /.*[Ss][Ii][Tt][Ee].*\n/gm);
+    removeReference(".prettierignore", /.*[Ss][Ii][Tt][Ee].*\n/gm);
+    removeReference("copy-schema-files.js", /.*[Ss][Ii][Tt][Ee].*\n/gm);
+    removeReference("lint-staged.config.js", /.*[Ss][Ii][Tt][Ee].*\n/gm);
+    removeReference("admin/src/environment.ts", /, "SITES_CONFIG"/g);
+    removeReference("./package.json", / browser:site/g);
+    removeReference("./package.json", /.*[Ss][Ii][Tt][Ee].*\n/gm);
+    removeReference("admin/src/config.ts", /.*[Ss][Ii][Tt][Ee].*\n/gm);
+    removeReference("dev-pm.config.js", /{[\n ]*name: "[Ss][Ii][Tt][Ee].*},\n/gs);
```

Comment:
you can use the ignore case modifier instead:
```suggestion
    removeReference("dev-pm.config.js", /{[\n ]*name: "site.*},\n/gsi);
```